src/use-swr.ts

```diff
@@ -507,41 +559,16 @@ function useSWR<Data = any, Error = any>(
       CACHE_REVALIDATORS[key].push(onUpdate)
     }
```

Comment:
We can move https://github.com/vercel/swr/pull/476/files#diff-ab20307c185620c9b54994fa4dcee1b7R512-R513 here and change https://github.com/vercel/swr/pull/476/files#diff-ab20307c185620c9b54994fa4dcee1b7R556-R560 to

```js
addRevalidator(RECONNECT_REVALIDATORS, onUpdate)
```